source/MaterialXView/Viewer.cpp

```diff
@@ -905,6 +908,47 @@ void Viewer::createAdvancedSettings(Widget* parent)
     {
         _bakeTextureRes = MIN_TEXTURE_RES * (int) std::pow(2, index);
     });
+
+    ng::Label* wedgeLabel = new ng::Label(advancedPopup, "Wedge Rendering Options");
+    wedgeLabel->setFontSize(20);
+    wedgeLabel->setFont("sans-bold");
+
+    ng::Widget* wedgeMin = new ng::Widget(advancedPopup);
+    wedgeMin->setLayout(new ng::BoxLayout(ng::Orientation::Horizontal));
+    mx::UIProperties wedgeProp;
+    wedgeProp.uiSoftMin = mx::Value::createValue(0.0f);
+    wedgeProp.uiSoftMax = mx::Value::createValue(1.0f);
+    ng::FloatBox<float>* wedgeMinBox = createFloatWidget(wedgeMin, "Minimum Wedge Multiplier:",
```

Comment:
What is the motivation for the term "multiplier" in these option names?  I would expect them to be named "Wedge Minimum Value" and "Wedge Maximum Value", but I may be missing the intent here.

Reply:
This is left over from the original intent that this be a multiplier on the value but didn't end up making sense. This should indeed be the absolute value and will make the change.